pkg/noun/jets/tree.c

```diff
@@ -2139,6 +2139,7 @@ static c3_c* _k140_ha[] = {
 
 /* new jets
 */
+  static u3j_harm _139_two_mate_a[] = {{".2", u3wb_mate, c3y}, {}};
```

Comment:
Style nit: This line should go on line ~`2274` (in between `_139_two__in_d` and `_139_two_d`)

Reply:
Done, thanks!